cross/arm64_ci_script.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+function exit_with_error {
+    set +x
+
+    local errorMessage="$1"
+
+    echo "ERROR: $errorMessage"
+    exit 1
+}
+
+#Exit if input string is empty
+function exit_if_empty {
+    local inputString="$1"
+    local errorMessage="$2"
+
+    if [ -z "$inputString" ]; then
+        exit_with_error "$errorMessage"
+    fi
+}
+
+# Cross builds corefx using Docker image
+function cross_build_native_with_docker {
+    __currentWorkingDirectory=`pwd`
+
+    # choose Docker image
+    __dockerImage=" hseok82/dotnet-buildtools-prereqs:ubuntu-16.04-crossx86-ef0ac75-20175511035548"
```

Comment:
I don't think we should be using `hseok82`, instead we should be using official dotnet-builtools images.

/cc @michaelsimons

Reply:
The Dockerfile for the image can be checked into https://github.com/dotnet/dotnet-buildtools-prereqs-docker.  Once this is done, it will get build and pushed to microsoft/dotnet-buildtools-prereqs